devicemapper/dmsetup_client.go

```diff
@@ -21,19 +21,26 @@ import (
 	"github.com/golang/glog"
 )
 
-// DmsetupClient is a low-level client for interacting with devicemapper via
-// the dmsetup utility.
+// DmsetupClient is a low-level client for interacting with device mapper via
+// the `dmsetup` utility, which is provided by the `device-mapper` package.
```

Comment:
Can you update the comments? Actually, having a note about the package in the global `Readme.md` might be super helpful. Not necessary for this PR though.